charts/castai-agent/templates/deployment.yaml

```diff
@@ -4,6 +4,9 @@ metadata:
   name: {{ include "castai-agent.fullname" . }}
   namespace: {{ .Release.Namespace }}
   labels:
+    {{ if .Values.global }}
```

Comment:
should this be `Values.global.azure` ?

Reply:
If I change it to `Values.global.azure` for our casual, not Azure K8s Extension installation, I get nil pointer exception.

```executing "castai-agent/templates/deployment.yaml" at <.Values.global.azure>: nil pointer evaluating interface {}.azure```

Reply:
Honestly, I put `Values.global.azure` as it is a requirement for an azure extension to work to have this naming convention.